src/components/Audio/Audio.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Video from '../Video';
+
+/**
+ * A component representing a Cloudinary served audio
+ * This component extends the Video component and replaces it's render function.
+ * Audio files are uploaded to Cloudinary as a video asset type.
+ * An <audio> tag with a video source, will play the audio only.
+ */
+class Audio extends Video {
+  /**
+   * Render an audio element
+   */
+  render() {
+    const {innerRef, fallback, children} = this.props;
+
+    const {
+      tagAttributes, // Attributes of this video element
+      sources        // <source> tags of this video element
```

Comment:
@maoznir , what are the default sources?

Reply:
good point, how about aac,mp3,wav?

Reply:
was set to aac,mp3,ogg